docs/client_reference.rst

```diff
@@ -697,6 +697,9 @@ TCPConnector
       *ssl_context* may be used for configuring certification
       authority channel, supported SSL options etc.
 
+  :param tuple local_addr: tuple of ``(local_addr, local_port)`` used to bind
```

Comment:
Don't use markup here but update ./docs/client_reference.rst also.
`(local_addr, local_port)` should be `(local_host, local_port)` pair.